frontend/apps/web/app/api/accounts/[accountId]/connections/postgres/check/route.ts

```diff
@@ -18,22 +18,24 @@ import { NextRequest, NextResponse } from 'next/server';
 export async function POST(req: NextRequest): Promise<NextResponse> {
   return withNeosyncContext(async (ctx) => {
     const body = (await req.json()) ?? {};
-    const db = await POSTGRES_CONNECTION.validate(body.db ?? {});
-    const tunnel = await SSH_TUNNEL_FORM_SCHEMA.validate(body.tunnel ?? {});
 
-    const pgconfig = new PostgresConnectionConfig({
-      connectionConfig: {
+    let pgconfig = new PostgresConnectionConfig({});
```

Comment:
i went with `let` since the properties are being reassigned later in the code (line 25, 31) vs. using `const` which, at least to me, indicates that the object isn't being reassigned at all (object or properties) even if technically you can change the properties, but if you'd prefer const, I can update it.